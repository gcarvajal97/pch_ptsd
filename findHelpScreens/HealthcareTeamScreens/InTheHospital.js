import React, { Component } from 'react';
import { View, Text, StyleSheet } from 'react-native';
import translate from '../../components/translateService';

export default class InTheHospital extends Component {
  render() {
    return (
      <View>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_p1')}</Text>

        <Text style={styles.paragraphTitle}>{translate('healthcareTeam.CH2_TITLE_1')}</Text>

        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t1_n1')}</Text>            
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t1_n2')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t1_n3')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t1_n4')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t1_n5')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t1_n6')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t1_n7')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t1_n8')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t1_n9')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t1_n10')}</Text>
            
        <Text style={styles.paragraphTitle}>{translate('healthcareTeam.CH2_TITLE_2')}</Text>

        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t2_n1')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t2_n2')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t2_n3')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t2_n4')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t2_n5')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t2_n6')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t2_n7')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t2_n8')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t2_n9')}</Text>

        <Text style={styles.paragraphTitle}>{translate('healthcareTeam.CH2_TITLE_3')}</Text>

        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t3_n1')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t3_n2')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t3_n3')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t3_n4')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t3_n5')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t3_n6')}</Text>
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t3_n7')}</Text>

        <Text style={styles.paragraphTitle}>{translate('healthcareTeam.CH2_TITLE_4')}</Text>

        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t4_p1')}</Text>
        
        <Text style={styles.paragraphBold}>{translate('healthcareTeam.Ch2_t4_b1')}</Text>
        <Text style={styles.paragraphBold}>{translate('healthcareTeam.Ch2_t4_b2')}</Text>
        <Text style={styles.paragraphBold}>{translate('healthcareTeam.Ch2_t4_b3')}</Text>
        
        <Text style={styles.paragraph}>{translate('healthcareTeam.Ch2_t4_p2')}</Text>

      </View>
    );
  }
}

const styles = StyleSheet.create({
  paragraphTitle: {
    fontWeight: 'bold',
    fontSize: 20,
    alignSelf: 'stretch',
    textAlign: 'left',
    marginTop: 6,
    marginBottom: 3,
    color: '#2089DC'
  },
  paragraph: {
    textAlign:'left',
    fontSize: 14,
    marginVertical:2,
  },
  paragraphBold: {
    textAlign:'left',
    fontSize: 14,
    fontWeight:'bold',
    marginVertical:2,
    marginLeft:10
  }
});