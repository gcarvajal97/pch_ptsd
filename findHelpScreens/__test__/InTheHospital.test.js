import React from 'react';
import InTheHospital from '../HealthcareTeamScreens/InTheHospital';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<InTheHospital/>).toJSON();
    expect(tree).toMatchSnapshot();
}) 