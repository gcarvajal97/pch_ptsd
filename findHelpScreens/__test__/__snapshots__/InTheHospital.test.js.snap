// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Aside from the injury itself, there are things about just being in the hospital that can be traumatic for children and parents.
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontFamily": "avenir-heavy",
        "fontSize": 20,
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    Working with the Healthcare Team
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    1. Don't be afraid to ask questions. Some parents keep a pad of paper to write down their questions. This way, when the health care team comes to examine your child, you will be prepared. If you don't understand the answer, let the medical team know and ask, again. Ask your doctor to write down the name of your child's injuries.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    2. Stay overnight with your child, if possible. Ask permission to be with your child during medical tests and procedures, if you want to be.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    3. Be sure to let the medical team know of any pre-existing medical conditions, other special needs and fears, and allergies your child has, as well as whatever you can tell them about how the injury happened.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    4. You are your child's most important advocate – ask how you can get involved in your child's care and when you can make decisions about treatment. Let your health care team know if your child is in pain.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    5. Take time for yourself. It is important to take breaks as often as you need them. Take a shower; make some phone calls; get something to eat or drink; or go to the chapel to think or cry – whatever it is that you need for yourself. After your break, you will feel better able to support your child. Let your child's health care team know when you leave your child's bedside and when you return.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    6. Make of list of what other people can do to help you. When people call and offer support, you will have something concrete to ask. You do not need to do it all.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    7. Ask the health care team to help you schedule all of your outpatient visits before you leave the hospital.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    8. Make sure that your child's pain is under control before you leave the hospital.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    9. Make sure that you have a good phone number to call if you have questions when you return home.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    10. Ask for copies of your child's x-rays, CT scans, or MRI scans.
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontFamily": "avenir-heavy",
        "fontSize": 20,
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    Tips for supporting your child in the hospital.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    1. You are the best person to help your child.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    2. Be patient with your child.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    3. Help your child understand what is happening.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    4. Make sure that you have a good phone number to call if you have questions when you return home.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    5. Talk about your feelings together.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    6. Help your child see the staff as helpers.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    7. Stay with your child (or have a family member or friend stay) as much as possible.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    8. Take care of yourself.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    9. Bring things from home to make your child comfortable – bath robe, toys, and favorite books.
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontFamily": "avenir-heavy",
        "fontSize": 20,
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    Tips for supporting your teen in the hospital.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    1. Be honest with your teen.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    2. Include your teen in medical discussions when possible.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    3. Encourage your child's health care team to talk with your teen directly and let him answer his own questions. He might want to talk alone with the doctors.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    4. Remember that teens can be self-conscious, and they may worry about how they will look and fit in with others, and about their privacy.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    5. Help your teen stay connected with old friends, as well as make new friends in the hospital.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    6. Bring things from home to make your child comfortable – bath robe, music, and magazines.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    7. Call your child's school to let them know what has happened.
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontFamily": "avenir-heavy",
        "fontSize": 20,
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    Resources for Families
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Most hospitals have staff who is experienced with helping other parents and caregivers in your situation. If you are upset, have questions about how to help your child, or just need to talk, please seek out someone at the hospital who is available to assist you and your family. This may be a:
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-heavy",
        "fontSize": 14,
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Social Worker
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-heavy",
        "fontSize": 14,
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Chaplain
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-heavy",
        "fontSize": 14,
        "marginLeft": 10,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Or a mental health professional, such as a psychologist or psychiatrist
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Once your child has been discharged and you have returned home, come back to this website for more information on ways parents can help their kids.
  </Text>
</View>
`;
